app/views/document_images/edit.html.erb

```diff
@@ -25,7 +27,7 @@
         },
         name: "alt_text",
         value: @image.alt_text,
-        error_message: @errors&.fetch("alt_text", nil)&.join("<br/>"),
+        error_message: alt_text_errors&.any? ? alt_text_errors.join("<br/>") : nil,
```

Comment:
Super minor and hopefully I didn't say this before but can't this just be `alt_text_errors&.join("<br>")` as that'll return nil?

Reply:
Not quite - if there are no errors, we need to return nil instead of an empty string.

Reply:
ooh good edge case catch 👍

Reply:
And now it's redundant with the update to publishing components :-).